api/src/services/graphql/index.ts

```diff
@@ -3232,6 +3244,81 @@ export class GraphQLService {
 			});
 		}
 
+		if ('directus_permissions' in schema.read.collections) {
+			schemaComposer.Query.addFields({
+				permissions_me: {
+					type: schemaComposer.createScalarTC<CollectionAccess>({
+						name: 'test',
```

Comment:
Should this be something like
```suggestion
						name: 'permissions_me_type',
```
?

Reply:
Oops forgot to rename the test type 🙈